src/Core/Rexl.Bind/Operations/Funcs/Text.cs

```diff
@@ -758,3 +758,66 @@ public static string Exec(string src, string remove, string insert)
         return src.Replace(remove, insert);
     }
 }
+
+public sealed partial class TextPadFunc : RexlOper
+{
```

Comment:
As suggested elsewhere:
* Perhaps have a `PadCenter` with alias `Pad` that adds to both sides.
* Use `Start` and `End` instead of `Left` and `Right`.

Reply:
Also register `PadCenter` as just `Pad` or perhaps make `Pad` the main name for it and make `PadCenter` be an alias?